eng/testing/performance/performance-setup.sh

```diff
@@ -264,7 +276,21 @@ if [[ "$wasm_runtime_loc" != "" ]]; then
     using_wasm=true
     wasm_dotnet_path=$payload_directory/dotnet-wasm
     mv $wasm_runtime_loc $wasm_dotnet_path
-    extra_benchmark_dotnet_arguments="$extra_benchmark_dotnet_arguments --wasmMainJS \$HELIX_CORRELATION_PAYLOAD/dotnet-wasm/runtime-test.js --wasmEngine /home/helixbot/.jsvu/$javascript_engine --customRuntimePack \$HELIX_CORRELATION_PAYLOAD/dotnet-wasm"
+    # install emsdk, $source_directory/src/mono/wasm/ has the nuget.config with require feed. EMSDK may be available in the payload in a different directory, should visit this install to avoid deplicated payload.
+    pushd $source_directory/src/mono/wasm/
+    make provision-wasm
+    EMSDK_PATH = $source_directory/src/mono/wasm/emsdk
```

Comment:
This variable gets set, but it does not ever seem to get used anywhere. What is it for?

Reply:
Yes, EMSDK_PATH is not used in our script file. It is set here is per document guidance at https://github.com/dotnet/runtime/blob/d953229d5429d2ffde833740dd481aab864d3e0c/src/mono/wasm/README.md